package.json

```diff
@@ -26,29 +28,50 @@
       "latest"
     ],
     "plugins": [
-      "transform-class-properties"
+      "transform-class-properties",
+      "transform-object-rest-spread",
+      [
+        "module-resolver",
+        {
+          "root": [
+            "./src"
+          ]
+        }
+      ]
     ]
   },
   "eslintConfig": {
     "extends": "react-app"
   },
-  "dependencies": {
-    "@kadira/storybook": "^2.5.2",
-    "axios": "^0.15.3",
+  "devDependencies": {
     "babel-cli": "^6.22.2",
+    "babel-plugin-module-resolver": "^2.7.0",
     "babel-plugin-transform-class-properties": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
     "babel-preset-latest": "^6.22.0",
     "babel-preset-react": "^6.22.0",
+    "react-scripts": "^0.8.4"
+  },
+  "dependencies": {
+    "axios": "^0.15.3",
+    "chart.js": "^2.5.0",
+    "element-resize-event": "^2.0.9",
     "eslint": "^3.16.1",
+    "faker": "^4.1.0",
+    "format-number": "^2.0.2",
+    "hex-rgba": "^1.0.1",
+    "jwt-simple": "^0.5.1",
     "lodash": "^4.16.2",
     "react": "^15.4.1",
+    "react-chartjs-2": "^2.0.5",
     "react-dom": "^15.4.1",
     "react-icons": "github:eggheadio/react-icons#3.0.2",
     "react-markdown": "^2.4.5",
-    "react-scripts": "^0.8.4",
-    "react-storybook-addon-info": "github:eggheadio/react-storybook-addon-info#4.0.4",
-    "react-storybook-addon-props-combinations": "^0.3.0",
+    "react-paginate": "^4.3.1",
+    "react-router": "^4.1.1",
+    "react-router-dom": "^4.1.1",
     "react-tabs": "^0.8.2",
+    "sort-by": "^1.2.0",
```

Comment:
Moving in dependencies from IC. Although these are fairly small deps, the code splitting in webpack 2 (in CRA already) should make this less of a problem